src/pages/p/[handle]/safe/index.page.tsx

```diff
@@ -0,0 +1,56 @@
+import { AppWrapper } from 'components/common'
+import Loading from 'components/Loading'
+import { ProjectSafeDashboard } from 'components/ProjectSafeDashboard'
+import { V1ProjectContext } from 'contexts/v1/projectContext'
+import { ProjectMetadataV5 } from 'models/project-metadata'
+import { GetStaticPaths, GetStaticProps, InferGetStaticPropsType } from 'next'
+import { V1UserProvider } from 'providers/v1/UserProvider'
+import { V1ProjectMetadataProvider } from 'providers/v1/V1ProjectMetadataProvider'
+import { V1ProjectProvider } from 'providers/v1/V1ProjectProvider'
+import { useContext } from 'react'
+import { getV1StaticPaths, getV1StaticProps } from '../pageLoaders'
+
+export const getStaticPaths: GetStaticPaths = async context => {
+  return getV1StaticPaths(context)
+}
+
+export const getStaticProps: GetStaticProps<{
+  metadata: ProjectMetadataV5
+  handle: string
+}> = async context => {
+  return getV1StaticProps(context)
+}
```

Comment:
**issue** I don't think we need static paths/props for the safe dashboard.

I think we only need a single `getServerSideProps` instead. Correct me if im wrong 🙏

Reply:
Nup, you are right! Thanks